tests/acceptance/features/coreApiShareManagementToShares/acceptShares.feature

```diff
@@ -51,7 +51,7 @@ Feature: accept/decline shares coming from internal users
       | /FOLDER/       |
       | /PARENT/       |
       | /textfile0.txt |
-    But user "Carol" should not see the following elements
+    And user "Carol" should not see the following elements
```

Comment:
It should be OK to have multiple `But` in a `Then` section.
https://github.com/gherlint/gherlint/issues/109

Can we get gherlint adjusted so that it allows this? And then run the newer linter so that these lines do not get adjusted.

Reply:
Yes, until and unless they are not sequential I think it should be okay to have multiple **But** in a **Then** section.

@saw-jan what do you think?

Reply:
@phil-davis for now, should i change the And step to But and push it again?

Reply:
Let's see if @saw-jan and @ishabaral can adjust Gherlint "soon", then we can avoid this problem.

Reply:
let me see

Reply:
new version released with some feature and bug fixes: https://github.com/gherlint/gherlint/releases/tag/v1.1.0